src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,15 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the following format: a/BLOCK, STREET, UNIT, POSTAL_CODE";
```

Comment:
how/where are you validating if the address has all 4 components? may be the program will crash with array index out of bound exception when you try to access some component that is not present, no?